jbpm-form-modeler-panels/jbpm-form-modeler-editor/jbpm-form-modeler-editor-backend/src/main/java/org/jbpm/formModeler/panels/modeler/backend/indexing/FormIndexVisitor.java

```diff
@@ -21,9 +21,9 @@
 import org.jbpm.formModeler.api.model.Field;
 import org.jbpm.formModeler.api.model.FieldType;
 import org.jbpm.formModeler.api.model.Form;
+import org.kie.workbench.common.services.refactoring.Resource;
+import org.kie.workbench.common.services.refactoring.ResourceReference;
```

Comment:
@manstis Could you please explain for us uninitiated what's the difference between the two and when should the first one be used over the other?